config/form_profile_mappings/22-5495.yml

```diff
@@ -1,5 +1,3 @@
-veteranDateOfBirth: [identity_information, date_of_birth]
-veteranFullName: [identity_information, full_name]
-veteranSocialSecurityNumber: [identity_information, ssn]
-toursOfDuty: [military_information, tours_of_duty]
-currentlyActiveDuty: [military_information, currently_active_duty]
```

Comment:
Same as above, the question about active duty is aimed at the applicant on the paper form.